web-ui/src/main/resources/catalog/components/elasticsearch/EsService.js

```diff
@@ -92,13 +95,26 @@
           if (luceneQueryString) {
             queryStringParams.push(luceneQueryString);
           }
-          queryHook.must.push({
-            query_string: {
-              query: queryStringParams.join(' AND ').trim()
-            }
-          });
-        }
 
+          if (titleOnly) {
```

Comment:
Here you bypass all other criteria coming from facets for example so we can't combine title only with a facet. I will fix that in another PR.

Reply:
https://github.com/geonetwork/core-geonetwork/pull/5911/commits/90dfd93d749ef1fb874713174f336c5ec764bddc